### PR TITLE
feat(angular): Add Angular 9 support + disable Ivy renderer due to bugs

### PR DESCRIPTION
**Pull request by dennisameling**
Refers to #19587

This PR introduces basic support for Angular 9, but doesn't compile the library with Ivy yet. Existing Ionic applications (e.g. with Angular 8) will continue to work with this PR, while actual support for Ivy is being worked on in https://github.com/ionic-team/ionic/tree/angular-ivy

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] **(n/a for Angular)** Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`npm run build`) was run locally and any changes were pushed
- [x] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: #19587

## What is the new behavior?

- Add Angular 9 support
- Disable Ivy renderer due to bugs (see the https://github.com/ionic-team/ionic/tree/angular-ivy for Ivy support)
-

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: angular/test/test-app/angular.json [hunk 1]
```diff
@@ -127,12 +144,16 @@
...
           }
         }
       }
     }
   },
-  "defaultProject": "test-app"
-}
+  "defaultProject": "test-app",
+  "cli": {
+    "analytics": "de9ff8d8-b1cf-40ad-9a28-80e7e8755d8e"
```
Comment (SchnWalter) [inline angular/test/test-app/angular.json]: Was this added intentionally?

Comment (dennisameling) [reply angular/test/test-app/angular.json]: No, apologies. This is Angular CLI anonymous tracking (see https://angular.io/analytics) that is automatically offered when using CLI. Pusing a new commit now that disables it.

File: angular/test/test-app/src/main.server.ts [hunk 2]
```diff
@@ -9,3 +9,5 @@ if (environment.production) {
 export { AppServerModule } from './app/app.server.module';
 export { ngExpressEngine } from '@nguniversal/express-engine';
 export { provideModuleMap } from '@nguniversal/module-map-ngfactory-loader';
+
+export { renderModule, renderModuleFactory } from '@angular/platform-server';
```
Comment (KillerCodeMonkey) [inline angular/test/test-app/src/main.server.ts]: please add a missing new line at the eof

Comment (dennisameling) [reply angular/test/test-app/src/main.server.ts]: Done :)

**Comment by mhartington** [comment]
Hello there.

https://github.com/ionic-team/ionic/pull/19515

This is the goal of #19515. I'm going to close this then.

**Pull request closed without merging**